### PR TITLE
GetAddressBalance fix when address is not indexed 

### PR DESCRIPTION
**Pull request by noescape00**

### CODE REVIEW

File: src/Stratis.Features.FederatedPeg/CollateralChecker.cs [hunk 1]
```diff
@@ -155,6 +155,7 @@ private async Task<bool> UpdateCollateralInfoAsync(CancellationToken cancellatio
 
             if (collateral.Count != addressesToCheck.Count)
             {
+                this.logger.LogDebug($"Expected {addressesToCheck.Count} data entries but received {collateral.Count}.");
```
Comment (dangershony) [inline src/Stratis.Features.FederatedPeg/CollateralChecker.cs]: String interpolation?

Comment (fassadlr) [reply src/Stratis.Features.FederatedPeg/CollateralChecker.cs]: Fixed

Comment (fassadlr) [reply src/Stratis.Features.FederatedPeg/CollateralChecker.cs]: Sorry this will be fixed another PR

Comment (dangershony) [reply src/Stratis.Features.FederatedPeg/CollateralChecker.cs]: ok so push when ready

**Pull request merged**